src/base/utils/foreignapps.cpp

```diff
@@ -77,7 +77,8 @@ namespace
                 return false;
             }
 
-            LogMsg(QCoreApplication::translate("Utils::ForeignApps", "Python detected, version: %1").arg(info.version), Log::INFO);
+            LogMsg(QCoreApplication::translate("Utils::ForeignApps", "Python detected, name: '%1', version: %2")
```

Comment:
"Name" is confusing to me. Why don't you write "Executable name"?

Reply:
will change